DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/loop_carried_dependency/src/CMakeLists.txt

```diff
@@ -4,16 +4,17 @@
 set(SOURCE_FILE loop_carried_dependency.cpp)
 set(TARGET_NAME loop_carried_dependency)
 set(EMULATOR_TARGET ${TARGET_NAME}.fpga_emu)
+set(SIMULATOR_TARGET ${TARGET_NAME}.fpga_sim)
 set(FPGA_TARGET ${TARGET_NAME}.fpga)
 
 # FPGA board selection
-if(NOT DEFINED FPGA_BOARD)
-    set(FPGA_BOARD "intel_a10gx_pac:pac_a10")
-    message(STATUS "FPGA_BOARD was not specified.\
-                    \nConfiguring the design to run on the default FPGA board ${FPGA_BOARD} (Intel(R) PAC with Intel Arria(R) 10 GX FPGA). \
+if(NOT DEFINED FPGA_TARGET)
```

Comment:
Was looking at this for updates to make for my FPGA sim PR and noticed this. FPGA_TARGET is already definied as ${TARGET_NAME}.fpga, would this not conflict with that?

Reply:
Yeah, that seems bad to me too.  I will wait for Ajay to complete his changes...